p/scripts/main.js

```diff
@@ -1340,14 +1344,28 @@ function closeNotification() {
 function init_notifications() {
 	notification = document.getElementById('notification');
 
-	notification.querySelector('a.close').onclick = function () {
+	notification.querySelector('a.close').addEventListener('click', function (ev) {
 		closeNotification();
+		ev.preventDefault();
 		return false;
-	};
+	});
+
+	notification.addEventListener('mouseenter', function () {
+		clearInterval(notification_interval);
+	});
+
+	notification.addEventListener('mouseleave', function () {
+		notification_interval = setTimeout(closeNotification, 1000);
+	});
```

Comment:
That seems like it'd be very easy to trigger accidentally. I'd say there's already the x if you want to get rid of it sooner.

Reply:
I know. 
Let me explain:
default timer to close the notification 4s or 8s.
When the user moves the mouse pointer over the notification: the timer will be stopped. So the user can read the message how long he/she needs. (Here the user can close the message box via the close icon.)
When the user moves the cursor out of the notification box, than it sets a new timer to 1 second.

Reply:
I realize that's the reasoning, but I don't think that addresses my remark. Your cursor can be in the general vicinity for any reason, and if you have arthritis your hand just shivers while operating the mouse even when you are completely aware of this behavior.

The long and short of it is that you're then ensuring the notification will disappear within one second after the cursor leaves the target. It should either reset to the full time (maybe minus one or two seconds, but not *to* one second) or track the actual starting time and resume.

Reply:
ah ok. Now I understood.
Will change it to 3 seconds.

Reply:
done